docs/shared/packages/angular/angular-plugin.md

```diff
@@ -77,6 +77,14 @@ Read more about:
 - [Library Types](/more-concepts/library-types)
 - [Buildable and Publishable Libraries](/more-concepts/buildable-and-publishable-libraries)
 
+### Using `@schematics/angular`
```

Comment:
I think the title should be something different, though I'm not sure what. Reading it made me think that this section describes how to use `@schematics/angular`, which is not the intention here. Devs can also generate code using the collection directly `nx g @schematics/angular:service`.

What we are trying to explain is that when a generator is run from the `@nrwl/angular` collection and it's not found, Nx falls back to look for it in the `@schematics/angular` collection.

Reply:
Good point.  Changed to "Fallback to `@schematics/angular`"